### ISSUE TITLE
Proposal: offer some reports on costs

### ISSUE THREAD

**Issue by amihaiemil** [zerocracy/farm#654]
0crat seems quite expensive: $4 per job is a lot, also considering that PDD generates a huge tree of tasks at least for the first version of a product. 

This being the case, clients have no idea what the cost of ZC actually is and it is also very hard to estimate it, since you don't have a clue how big and fast will the puzzle tree grow.

Proposal: 0crat should offer some reports on previously done projects (maybe with the approval of the POs), so prospective clients have an idea of the price.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@amihaiemil good idea. I think we should just make finances of sanbox projects open to everybody, in some demo mode. That will help clients (and programmers) understand how much the development costs.

**Comment by 0crat** [comment]
The job [#654](https://github.com/zerocracy/farm/issues/654) assigned to @yegor256[/z](https://www.0crat.com/u/yegor256), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/571960449). The budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 points just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), total is [+1070](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+10939](https://www.0crat.com/u/yegor256)

**Comment by amihaiemil** [comment]
@yegor256 I think we should also offer reports on bigger projects. Ask POs if we can make such reports on their projects. jpeek and cactoos are just libraries, not relevant for a big application :) the tree from jpeek is not relevant to the cost of zerocrat/farm, for instance.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) I see that you unassigned this issue; the order is still assigned to @yegor256[/z](https://www.0crat.com/u/yegor256) though; to cancel the order use `refuse`, as in [§6](http://www.zerocracy.com/policy.html#6)
